lib/servers/datavault/data_vault.py

```diff
@@ -1148,6 +1159,19 @@ def get(self, c, limit=None, startOver=False):
         dataset.keepStreaming(c.ID, c['filepos'])
         return data
 
+    ### Add in saving camera images as a .npy with the dataset
+
+    @setting(22, data='*i', image_size='*i', repetitions='i', returns = '')
+    def save_image(self, c, data, image_size, repetitions):
```

Comment:
One more comment, this should probably be changed to "save_npy_image" or "save_image_npy"

I would also update this docstring that this method save a .npy file.

I didn't catch that this was npy specific.